app/views/helpers/ChargeName.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.helpers
+
+import models.financialDetails.{BalancingCharge, PoaOneDebit, PoaTwoDebit, TransactionType}
+import play.api.i18n.Messages
+
```

Comment:
I think we should start unifying Lang specific converters: Type -> String etc
Maybe name like  ChargeNameLangHelper would be more intuitive, not sure, worth to create a tech story to start moving similar code under one namespace/helper?